templates/macros/your-tip.html

```diff
@@ -236,17 +236,17 @@ <h5 class="list-group-item-heading">{{ _("Manual renewal") }}</h5>
     % endif
     % if patron_visibilities == 1
     <p class="text-info">{{ glyphicon('info-sign') }} {{ _(
-        "{username} has chosen not to see who their patrons are, so your donation will be secret.",
```

Comment:
This is a known singular, where "their" would denote plural.

Reply:
Liberapay uses [singular they](https://en.wikipedia.org/wiki/Singular_they).

Reply:
Hence why I made this PR to fix it.